requirements.txt

```diff
@@ -21,4 +21,5 @@ decorator~=5.1.1
 urllib3[secure]
 semantic-version==2.10.0
 pysftp~=0.2.9
+ordered-set~=4.0.2
```

Comment:
Why we need ordered-set ?

Reply:
to work with sets but preserving the order of initial lists.
list1 = [2,3,5]
list2=[1,6,3]

OrderedSet(list1) - OrderedSet(list2) = (2,5)

native set does not care about the order